tmp/unrelated.txt

```diff
@@ -0,0 +1 @@
+foo
```

Comment:
Added by mistake?

Reply:
Certainly! I’m surprised there aren’t more given how many tempdirs I’ve burned in this process 😄 Thanks for catching that. Will fix!